rust/src/dcerpc/dcerpc_udp.rs

```diff
@@ -73,22 +73,29 @@ impl DCERPCUDPState {
     }
 
     pub fn free_tx(&mut self, tx_id: u64) {
-        SCLogDebug!("Freeing TX with ID {} TX.ID {}", tx_id, tx_id+1);
+        SCLogDebug!("Freeing TX with ID {} TX.ID {}", tx_id, tx_id + 1);
         let len = self.transactions.len();
         let mut found = false;
         let mut index = 0;
         for i in 0..len {
             let tx = &self.transactions[i];
-            if tx.id as u64 == tx_id { //+ 1 {
+            if tx.id as u64 == tx_id {
+                //+ 1 {
                 found = true;
                 index = i;
                 SCLogDebug!("tx {} progress {}/{}", tx.id, tx.req_done, tx.resp_done);
                 break;
             }
         }
         if found {
-            SCLogDebug!("freeing TX with ID {} TX.ID {} at index {} left: {} max id: {}",
-                            tx_id, tx_id+1, index, self.transactions.len(), self.tx_id);
+            SCLogDebug!(
```

Comment:
this way having `SCLogDebug!` statements becomes really disruptive wrt following the code flow. These need to be compact.

Reply:
We can prevent the SCLog* macros from any formatting at all.  But otherwise there are no options to fine tune this.

Reply:
Unfortunately the feature to disable formatting of macro calls by name is only available in nightly rust.  The nom macros get messed up as well, however we can put an ignore specifically before them, but thats a little tedious before our logging macros.

This behaviour does appear to be idiomatic for lines this long.. 5 of 5 random Rust projects I checked are `rustfmt --check` clean and do this.

Reply:
Another work-around to prevent formatting inside a macro..
```
SCLogDebug{"freeing TX with ID...
```

Reply:
I think having verbose debug statements would be an anti-pattern. They'd hinder the ease of following the program logic.

Reply:
@victorjulien It looks like our clang format has the same issue: https://github.com/OISF/suricata/blob/master/src/detect-file-data.c#L512